### PR TITLE
Place key points and some notes in the margin

### PR DESCRIPTION
**Pull request by hyanwong**
Also remove hidden comments about ARGs

These are all short and far enough away from code blocks that they should be OK on mobile (as discussed at #87)

### CODE REVIEW

**Review by jeromekelleher — APPROVED** [review]
LGTM

File: what_is.md [hunk 1]
```diff
@@ -249,19 +256,26 @@ plt.show()
 
 ## A record of genetic ancestry
 
+::::{margin}
```
Comment (benjeffery) [inline what_is.md:259]: This one collides with the "todo" but I think that is ok as it is a temporary problem.
Screenshot from 2021-06-22 10-26-11

**Review by benjeffery — APPROVED** [review]
These are great @hyanwong, perfect for skimmers getting an overview.

File: what_is.md (hunk 1 above)
Comment (hyanwong) [reply what_is.md:259]: Yep - I'm just sorting that now. Also they don't actually overlay each other, unlike the code boxes.

**Pull request merged**